reference/6/Microsoft.PowerShell.Utility/Test-Json.md

```diff
@@ -20,70 +20,89 @@ Test-Json [-Json] <string> [[-Schema] <string>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The Test-Json cmdlet tests whether a string is a valid JavaScript Object Notation (JSON) document and can optionally very that JSON document against a provided schema.
 
-The verified string can then be used with the ConvertFrom-Json cmdlet convert a JSON-formatted string to a JSON object, which is easily managed in PowerShell or sent to another program or web service that access JSON input.
+The Test-Json cmdlet tests whether a string is a valid JavaScript Object Notation (JSON) document
```

Comment:
Should the first instance of `Test-Json` also be in a codefence or is it excluded since this page is about it?  It's codefenced in other areas of the doc.

Reply:
fixed